fares-data-build-tool/fdbt-site
# Remove proof for adult and infant in group passenger types

# Description

For infant and adult in group passenger types the proof option is not required when defining the passengers.

# Testing instructions

- Select a faretype
- Once submitted change the url to http://localhost:5555/groupPassengerTypes
- Select adult, infant and continue
- for adult and infant the options should not show for proof
- Try the above again but select child and adult and the child should see the proof option


# Type of change


# Checklist:
